build.sbt

```diff
@@ -25,20 +31,54 @@ def jsonDependencies(scalaVersion: String) = Seq(
   logback % Test
 ) ++ jacksons ++ specsBuild.map(_ % Test)
 
+// Common settings 
+import com.typesafe.sbt.SbtScalariform._
+import scalariform.formatter.preferences._
+
+val previousVersion = Some("2.6.0-M1") // first from this separate repo
```

Comment:
I agree that's not a "normal" check: not thefirst version w/o a previous version, not a backward compatible release.

Even if this 2.6.0 will be a major version with some changes breaking the compat (such as the packages), I personnally prefer these breakage were "under control".

At least checking against M1, the API changes other than the packages renaming are validated (and other than what's ok to filter).